### ISSUE TITLE
Issues with voiceover and zebra

### ISSUE THREAD

**Issue by demhademha** [zbrateam/Zebra#1068]
**Describe the bug**
When using voiceover, voiceover cannot detect the toggle for turning on/off dark mode. Also, voiceover cannot read package installation info (when the console is running)

**To Reproduce**
Turn on voiceover and try turning the dark mode  toggle with your eyes closed  

**Expected behavior**
Expected voiceover to fully work 

**Screenshots**

**Device Information (please complete the following information):**
 - iOS Version [e.g. 12.1.2 or 10.3.3]: <13.3>
 - Zebra Version [e.g. 1.0\~beta1 or beta1]: <1.0.7>
 - Device Type [e.g. iPhone or iPad]: <ipad>
 - Jailbreak Type [e.g. unc0ver, Chimera, checkra1n, etc.]: <unc0ver but also happens on any other jailbreak>

**Crash Log or System Log**
Please provide a Crash Log to help us solve the issue.

**Additional context**
you might want to label the above elements so they are detected by voiceover

**Comment by wstyres** [comment]
As of 1.1, the dark mode button does not appear anymore on devices with iOS 13 and up, the issue still needs to be resolved for 12 and below.

**Comment by demhademha** [comment]
Was this issue resolved?

**Comment by wstyres** [comment]
After reading again no it isn't fully resolved. The dark mode button you mentioned has been removed but I have not tested to see if voiceover works on the console yet.

**Comment by iAdam1n** [comment]
Can you check if this is still an issue in the latest release please?
